### ISSUE TITLE
How to set the learning rate?

### ISSUE THREAD

**Issue by codenewww** [openai/improved-diffusion#84]
When I train the model for some steps,I often find that the loss value rise up to a very high level(such as 747),what happened?Is it related to the value of the lr?
